lucien-ios/LoginScreen/LoginViewModel.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  LoginManager.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 9/29/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+final class LoginViewModel {
+
+    let lucienAPIClient = LucienAPIClient()
+    var currentUser: User?
+    var dashboardData: Dashboard?
+    var hasCollection: Bool = false
+
+    func getCurrentUser(completion: @escaping () -> Void) {
+        lucienAPIClient.getCurrentUser { response in
+            switch response {
+            case .success(let user):
+                self.currentUser = user
+                completion()
+            case .failure(let error):
+                print(error)
+            }
+        }
+    }
+
+    func authenticateUser(code: String, completion: @escaping () -> Void) {
```

Comment:
This is okay to have a completion since we're not using Rx yet but once we do it shouldn't have one